src/ResourceManager/ApiManagement/Commands.ApiManagement/Commands/NewAzureApiManagementVirtualNetwork.cs

```diff
@@ -24,16 +24,6 @@ namespace Microsoft.Azure.Commands.ApiManagement.Commands
     [Cmdlet("New", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "ApiManagementVirtualNetwork"), OutputType(typeof(PsApiManagementVirtualNetwork))]
     public class NewAzureApiManagementVirtualNetwork : AzureRMCmdlet
     {
-        [Parameter(
-            ValueFromPipelineByPropertyName = false,
-            Mandatory = true,
-            HelpMessage = "Location of the virtual network.")]
-        [LocationCompleter("Microsoft.ApiManagement/service")]
-        [ValidateNotNullOrEmpty]
-        [Obsolete("The location property will be deprecated in future" +
-            "breaking change release. It is no longer required.")]
```

Comment:
@solankisamir just for clarification, is location no longer required (can be optionally provided), or is it no longer used (same result if the API call is given the location or not)?

Reply:
@cormacpayne the location was just been accepted from powershell, but never being passed to the actual .NET contract.